recipe/conda_build_config.yaml

```diff
@@ -332,8 +330,7 @@ gf2x:
 gsl:
   - 2.4
 gstreamer:
-  - 1.12  # [linux]
-  - 1.14.0  # [osx]
+  - 1.14.0
```

Comment:
isn't 1.14.4 the most recent build?

Reply:
Okay, maybe then I actually did not understand pinning. My thoughts here were:
- I pin 1.14.0 as build time dependency, because it is compatible to 1.14.0 - 1.14.4
- the run time dependency is pinned to 1.14.x in gstreamer's meta.yaml

If I would pin 1.14.4 here, and the runtime dependeny is set to 1.14.x it could be possible to have built with 1.14.4, but having 1.14.1 as run time, which in rare cases can lead to missing symbols.

Reply:
`1.14.4` and `max_pin=x.x` means `>=1.14.4,<1.15.0a0`

Reply:
@mingwandroid we actually discussed something very similar recently. Your opinion here may be very valuable.

Reply:
@isuruf @scopatz, so it seems that 1.14.4 would work. Shall we go for it then?

Reply:
Sure.